### PR TITLE
Fix crash after onboarding on fresh simulator instance

### LINKED ISSUE #1172: Crash after onboarding on fresh simulator instance
1. I went through onboarding in the simulator. The app was freshly installed and the device was just reset.
2. After onboarding I saw this screen:
Screenshot 2023-02-20 at 16 38 32
3. I clicked "Start Using Planetary".
4. Debugger was triggered and reported the following error: `2023-02-20 16:38:40.113499+0100 Planetary[51926:9940523] Swift/ContiguousArrayBuffer.swift:600: Fatal error: Index out of range`.

```
Thread 1 Queue : com.apple.main-thread (serial)
#0	0x000000018bfa78e0 in _swift_runtime_on_report ()
#1	0x000000018c034580 in _swift_stdlib_reportFatalErrorInFile ()
#2	0x000000018bce2e10 in closure #1 in closure #1 in closure #1 in _assertionFailure(_:_:file:line:flags:) ()
#3	0x000000018bce2b78 in closure #1 in closure #1 in _assertionFailure(_:_:file:line:flags:) ()
#4	0x000000018bce2564 in _assertionFailure(_:_:file:line:flags:) ()
#5	0x000000018bcc3eb8 in _ArrayBuffer._checkInoutAndNativeTypeCheckedBounds(_:wasNativeTypeChecked:) ()
#6	0x000000018bcc7bf0 in Array.subscript.getter ()
#7	0x0000000100f5d164 in OnboardingViewController.currentStep.getter at /Users/filip/repositories/planetary-ios-raptor/Source/Onboarding/OnboardingViewController.swift:47
#8	0x0000000100f6051c in OnboardingViewController.navigationController(_:didShow:animated:) at /Users/filip/repositories/planetary-ios-raptor/Source/Onboarding/OnboardingViewController.swift:178
#9	0x0000000100f605d0 in @objc OnboardingViewController.navigationController(_:didShow:animated:) ()
#10	0x00000001084237f8 in -[UINavigationController navigationTransitionView:didEndTransition:fromView:toView:] ()
#11	0x0000000108fa7280 in -[UINavigationTransitionView _notifyDelegateTransitionDidStopWithContext:] ()
#12	0x0000000108fa74ec in -[UINavigationTransitionView _cleanupTransition] ()
#13	0x0000000108fd8df0 in +[UIView(UIViewAnimationWithBlocks) conditionallyAnimate:withAnimation:layout:completion:] ()
#14	0x0000000108fa6f80 in -[UINavigationTransitionView transition:fromView:toView:] ()
#15	0x000000010842b8fc in -[UINavigationController _startTransition:fromViewController:toViewController:] ()
#16	0x000000010842bc58 in -[UINavigationController _startDeferredTransitionIfNeeded:] ()
#17	0x000000010842cab8 in -[UINavigationController __viewWillLayoutSubviews] ()
#18	0x00000001084142e0 in -[UILayoutContainerView layoutSubviews] ()
#19	0x0000000108fe1bc8 in -[UIView(CALayerDelegate) layoutSublayersOfLayer:] ()
#20	0x0000000187f18844 in CA::Layer::layout_if_needed(CA::Transaction*) ()
#21	0x0000000187f232fc in CA::Layer::layout_and_display_if_needed(CA::Transaction*) ()
#22	0x0000000187e51058 in CA::Context::commit_transaction(CA::Transaction*, double, double*) ()
#23	0x0000000187e7ce28 in CA::Transaction::commit() ()
#24	0x0000000108b715c8 in _afterCACommitHandler ()
#25	0x000000018037217c in __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ ()
#26	0x000000018036cb1c in __CFRunLoopDoObservers ()
#27	0x000000018036cfd4 in __CFRunLoopRun ()
#28	0x000000018036c7f4 in CFRunLoopRunSpecific ()
#29	0x0000000188faec98 in GSEventRunModal ()
#30	0x0000000108b485d4 in -[UIApplication _run] ()
#31	0x0000000108b4c5cc in UIApplicationMain ()
#32	0x0000000101060ae0 in main at /Users/filip/repositories/planetary-ios-raptor/Source/App/AppDelegate.swift:15
#33	0x0000000104441fa0 in start_sim ()
#34	0x00000001045c5e50 in start ()
```

Screenshot 2023-02-20 at 16 43 19

### PR DESCRIPTION
**Pull request by martindsq**
Closes #1172

### CODE REVIEW

File: Source/Configs/Planetary.debug.xcconfig [hunk 1]
```diff
@@ -55,7 +55,7 @@ PLANETARY = @l1sGqWeCZRA99gN+t9sI6+UOzGcHq3KhLQUYEwb4DCo=.ed25519
 PLANETARY_SYSTEM = ${PLANETARY}
 
 ALIAS_SERVER_ONE = Planetary Alias::net:planetary.name:8008~shs:42ACINb4+Qjd1kPYYpJ67V16CTHIWp93cjd1gDX7FKY=::icon-planetary-3::planetaryRoomToken
-PLANETARY_COMMUNITY_ALIAS_SERVERS = ${ALIAS_SERVER_ONE}
+PLANETARY_COMMUNITY_ALIAS_SERVERS = ${ALIAS_SERVER_ONE}||
```
Comment (boreq) [inline Source/Configs/Planetary.debug.xcconfig:58]: Is this correct?

**Comment by boreq** [comment]
I checked and it no longer crashes for me on this branch.

**Review by boreq — APPROVED** [review]

**Comment by boreq** [comment]
Merging since it works.

File: Source/Configs/Planetary.debug.xcconfig (hunk 1 above)
Comment (martindsq) [reply Source/Configs/Planetary.debug.xcconfig:58]: yes, it is. we have the same weird || in the release config, which is the definitive one. It is a way of telling the parsing algorithm that it is an array of one element, not just a string

**Pull request merged**